src/screen/ScoreScreen.java

```diff
@@ -240,14 +246,32 @@ private void saveCoin() {
         }
     }
 
+	/**
+	 * Saves the gem into currency file
+	 */
+	// CtrlS
+	private void saveGem() {
+		try {
+			Core.getCurrencyManager().addGem(1);
+			logger.info("You eared 1 Gem for Game Clear");
+		} catch (IOException e) {
+			logger.warning("Couldn't load gem!");
+		}
+	}
+
 	/**
 	 * Draws the elements associated with the screen.
 	 */
 	private void draw() {
 		drawManager.initDrawing(this);
 
-		drawManager.drawGameOver(this, this.inputDelay.checkFinished(),
-				this.isNewRecord);
+		if (this.livesRemaining > 0 && this.level > 7) {
+			drawManager.drawGameClear(this, this.inputDelay.checkFinished(),
+					this.isNewRecord);
+		} else {
+			drawManager.drawGameOver(this, this.inputDelay.checkFinished(),
+					this.isNewRecord);
+		} // CtrlS
```

Comment:
this.isGameClear 필드를 추가해, ScoreScreen 생성자에서 작성해주신 조건문을 활용해 isGameClear를 초기화 할 수 있을 것 같습니다.

수정된 drawGameEnd는 다음과 같이 isGameClear 필드를 추가해서 호출합니다.
```suggestion
		drawManager.drawGameEnd(this, this.inputDelay.checkFinished(), this.isNewRecord, this.isGameClear); // CtrlS
```

Reply:
불필요한 코드 라인 수 증가도 피하고 좋은 거 같습니다. 반영해보도록 하겠습니다.